common/configuration.ts

```diff
@@ -513,6 +519,8 @@ export const networkConfig: { [key: string]: INetworkConfig } = {
       sUSDbC: '0x4c80e24119cfb836cdf0a6b53dc23f04f7e652ca',
       wstETH: '0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452',
       STG: '0xE3B53AF74a4BF62Ae5511055290838050bf764Df',
+      USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
```

Comment:
just noting that basescan shows `agUSD` and not `USDA`. is this on purpose?

Reply:
If you look at the token symbol https://basescan.org/address/0x0000206329b97DB379d5E1Bf586BbDB969C63274#readProxyContract#F17, it's `USDA`. We'll check to update it, `agUSD` was used in the first version of the token and was subsequently renamed to `USDA`.

Reply:
ok i think you need to reach out to etherscan to get them to refresh it